lib/assets/javascripts/turbograft/remote.coffee

```diff
@@ -33,7 +33,7 @@ class TurboGraft.Remote
       triggerEventFor 'turbograft:remote:start', @initiator,
         xhr: xhr
 
-    xhr.addEventListener 'error', @onError
+    xhr.addEventListener 'error', @onError.bind(this)
```

Comment:
More CoffeeScript-ey solution is to use `=>` in the method definition (which binds the method in the constructor).